osu.Game/Online/OnlineViewContainer.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Online.API;
+using osu.Game.Online.Placeholders;
+
+namespace osu.Game.Online
+{
+    /// <summary>
+    /// A <see cref="Container"/> for dislaying online content who require a local user to be logged in.
+    /// Shows its children only when the local user is logged in and supports displaying a placeholder if not.
+    /// <see cref="PopIn"/> and <see cref="PopOut"/> methods can be overriden to customize fade in/out behaviour
```

Comment:
```suggestion
    /// <see cref="PopIn"/> and <see cref="PopOut"/> methods can be overriden to customize visibility animation.
```

Reply:
why is this necessary in the first place? that should be documented on `VisibilityContainer`, not this one.